nav2_amcl/src/amcl_node.cpp

```diff
@@ -75,6 +76,11 @@ AmclNode::AmclNode()
   declare_parameter("laser_max_range", rclcpp::ParameterValue(100.0));
   declare_parameter("laser_min_range", rclcpp::ParameterValue(-1.0));
   declare_parameter("laser_model_type", rclcpp::ParameterValue(std::string("likelihood_field")));
+  declare_parameter("set_initial_pose", rclcpp::ParameterValue(false));
+  declare_parameter("initial_pose.x", rclcpp::ParameterValue(0.0));
+  declare_parameter("initial_pose.y", rclcpp::ParameterValue(0.0));
+  declare_parameter("initial_pose.z", rclcpp::ParameterValue(0.0));
+  declare_parameter("initial_pose.angle", rclcpp::ParameterValue(0.0));
```

Comment:
`initial_pose_yaw` would be a better name.

Reply:
Ok, I'll update.

Reply:
Though if the others are `initial_pose.XYZ` for the position, if the angle doesnt have the `.` it doesn't match that parameterization style if angle is `_`.

Reply:
`initial_pose.yaw`?

Reply:
yup, works. I don't really care `.` or `_` as long as we're consistent with it

Reply:
Yes, I'll use initial_pose.yaw to be consistent with the others.